browsertime/visualmetrics-portable.py

```diff
@@ -414,34 +403,15 @@ def video_to_frames(
                 gc.collect()
                 if extract_frames(video, directory, full_resolution, viewport):
                     client_viewport = None
-                    if find_viewport and options.notification:
-                        client_viewport = find_image_viewport(
-                            os.path.join(directory, "video-000000.png"), is_mobile
-                        )
-                    if multiple and orange_file is not None:
-                        directories = split_videos(directory, orange_file)
-                    else:
-                        directories = [directory]
+                    directories = [directory]
                     for dir in directories:
-                        trim_video_end(dir, trim_end)
                         if orange_file is not None:
-                            remove_frames_before_orange(dir, orange_file)
```

Comment:
This method should stay or be combined with `remove_orange_frames`

Reply:
Ok, I can add that but how do that work on your side? Usually the orange is added first, then the recording is started?

Reply:
I've made the change but if I remember correctly the workflow is like this:
- Add the orange frame
- wait 
- start the video
 
I wonder than if that wait time is too short since it affected your tests, or is it another use case?

Reply:
But I think that do not matter now, I added it so I hope it works when you retry :) Then we can look at the logic later.